1_basics.ipynb

```diff
@@ -101,22 +189,47 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# Gazebo, Rvizの使い方"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
    "source": [
-    "## 1. Gazeboの使い方\n",
+    "# How to use Gazebo and Rviz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
+   "source": [
+    "## Gazeboの使い方\n",
     "gazeboはロボットのシミュレータです。以下の手順で起動します。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## How to use Gazebo\n",
+    "Gazebo is a robot simulator. Follow the steps below to start the simulation."
```

Comment:
The original is fine if there is a specific previously mentioned simulation that the text goes on to talk about.

If, instead, these are general instructions for starting any simulation, the following would be better:

    "Gazebo is a robot simulator. Follow the steps below to start **a** simulation."

Reply:
Thank you. It may not mentioned clearly yet in this specific notebook, but we will always refer to our WRS simulator in this series of notebooks. So "the" might be better.

Reply:
I might have been a bit unclear here.
You are having the user follow steps to start up a new simulation using Gazebo, yes?
If so, then "start a simulation." sounds better.

Reply:
Ahh, I confused the "simulator" and "simulation.

For simulator, it would be "the".

For simulation, it would be "a" (you are correct).